examples/all-clusters-app/linux/args.gni

```diff
@@ -29,3 +29,5 @@ matter_log_json_payload_decode_full = true
 matter_log_json_payload_hex = true
```

Comment:
Add blocking comment on https://github.com/project-chip/connectedhomeip/pull/31516 to be merged first